ddapm_test_agent/agent.py

```diff
@@ -591,6 +591,11 @@ def main(args: Optional[List[str]] = None) -> None:
             "to the agent."
         ),
     )
+    parser.add_argument(
+        "--uds-path",
+        type=str,
+        default=os.environ.get("DD_AGENT_URL") if os.environ.get("DD_AGENT_URL", "").startswith("unix://") else "/var/run/datadog/apm.socket"
```

Comment:
Figure out if to hard fail if default socket not available (most likely will by default)